<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="emailInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(37,39,48);
border-radius: 10px;
color: rgb(26, 26, 26);
padding-left: 20px;
padding-right:20px;
height: 40px;
width: 40px;
border-bottom: 2px solid rgb(37,39,48);
border-bottom-left-radius: 10px; 
border-bottom-right-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>200</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid rgb(37,39,48);
border-radius: 10px;
color: rgb(26, 26, 26);
padding-left: 20px;
padding-right:20px;
height: 40px;
width: 40px;
border-bottom: 2px solid rgb(37,39,48);
border-bottom-left-radius: 10px; 
border-bottom-right-radius: 10px;
}</string>
   </property>
   <property name="placeholderText">
    <string>Passwort</string>
   </property>
  </widget>
  <widget class="QLabel" name="OutputName">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
padding-left: 20px;
padding-right:20px;
height: 40px;
width: 40px;s
}</string>
   </property>
   <property name="text">
    <string>Sign In</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogInButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegisterButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>201</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="verlassenButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>verlassen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
